sdk/python/kfp/dsl/_container_op.py

```diff
@@ -268,5 +269,38 @@ def add_node_selector_constraint(self, label_name, value):
     self.node_selector[label_name] = value
     return self
 
+  def add_pod_annotation(self, name: str, value: str):
+    """Adds a pod's metadata annotation.
+
+    Args:
+      name: The name of the annotation.
+      value: The value of the annotation.
+    """
+
+    self.pod_annotations[name] = value
+    return self
+
+
+  def set_tpu(self, tpu_cores: str, tpu_resource: str, tf_version: str):
```

Comment:
can we move it to `kfp.gcp`?

     from kfp.gcp import use_tpu
     ContainerOp(...).apply(use_tpu(tpu_cores=4))

Reply:
done